dftimewolf/lib/collectors/gsheets.py

```diff
@@ -222,22 +225,22 @@ def _ParseSheet(self, spreadsheet_id: str, sheet_title: str) -> DataFrame:
     # Removing white spaces from column names
     df.rename(columns=lambda name: name.strip(), inplace=True)
 
-    for column in self._mandatory_columns:
-      if column not in df.columns:
-        self.logger.error(
-            'Mandatory column "{0:s}" was not found in sheet "{1:s}".'.format(
-                column, sheet_title))
-        self.logger.error('Please make sure all mandatory are present:')
-        self.logger.error(
-            '"message": String with an informative message of the event')
-        self.logger.error(
-            '"datetime": ISO8601 format for example: 2015-07-24T19:01:01+00:00')
-        self.logger.error(
-            '"timestamp_desc": String explaining what type of timestamp it is for example file created'
-        )
-        return None
+    if self._validate_columns:
```

Comment:
Use a guard statement here instead of adding an extra nested block (and save an indentation level).

```python
if not self._validate_columns:
  return df
 
for column in self._mandatory_columns:
# ...
```

Reply:
Done.